internal/cli/cli.go

```diff
@@ -54,66 +57,66 @@ func main() {
 		}
 
 		// Data Field Header
-		dataHeader := TMDataFieldHeader{}
+		dataHeader := pus.TMDataFieldHeader{}
 		dhsize := binary.Size(dataHeader)
 		dhr := bytes.NewReader(buf[0:])
-		err = dataHeader.read(dhr)
+		err = dataHeader.Read(dhr)
```

Comment:
can it not just be `reader` instead of another abbreviation (`dhr`) or just pass the reader directly as argument if it's not needed afterwards.